tdrs-frontend/src/mirage.js

```diff
@@ -8,11 +14,25 @@ export default function startMirage(
     environment,
 
     routes() {
-      this.namespace = 'mock_api'
+      this.urlPrefix = 'http://localhost:8080/v1'
+      this.post('/reports/', () => 'Success')
+      this.get('/auth_check/', () => {
+        if (
+          window.localStorage.getItem('loggedIn') ||
```

Comment:
If we are using pa11y, we want to mock the auth always. If we are using the mock api for development, we want to be able to log in and out